Makefile

```
@@ -4,6 +4,7 @@ build-book:
 	# list folders with notebooks here. Notebooks must be present in _toc.yml.
 	cp -r infrastructure book/
 	cp -r vector_data book/code
+	cp -r imagery book/code
```

```suggestion
	cp -r imagery/* book/code
```

This will also fix the issue with #25.